docs/guides/how-to-release-the-components.md

```diff
@@ -3,45 +3,23 @@ kind: 📌 Docs
 ---
 # How to release the components?
 
-Here's the guide to create a new release:
+## Release phase
 
-1. Make sure you updated the `CHANGELOG.md` with all the new features and bugfixes.
-1. Decide if you're doing a `major`, `minor` or `patch` version or even a `prerelease` according to the semver spec.
-1. Make sure the commit you want to version is on `master`.
-1. Run the `npm version` command with the right parameters (see details below).
-  * This will update `package.json` and `package-lock.json` in a new commit.
-  * This will create a git tag.
-1. Push the updated `master` branch with the new tag on GitHub (see details below).
-1. Run `npm publish` to deploy the new version on npmjs.com (see details below).
+Release is automatized with [release-please](https://github.com/googleapis/release-please).
```

Comment:
"automatized" exists but it's not used a lot, should probably use "automated" instead